sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -362,7 +362,7 @@ case class Join(
         left.constraints
       case RightOuter =>
         right.constraints
-      case FullOuter =>
+      case _ =>
```

Comment:
nit: for readability, how about leaving the comment like this?
```
case _ => // FullOuter
```

Reply:
Ya. I thought like that, but actually, there are many missing patterns.

Reply:
It was difficult to track one-by-one. Logically, this is the rest of the previous patterns. So, I decided to skip that comment.